### DISCUSSION TITLE
Questions regarding .env

### DISCUSSION THREAD

**Discussion by jaschaurbach** [bookwyrm-social/bookwyrm#2824] [General]
in .env there is

`DEFAULT_LANGUAGE="English"`

My question: In which language does this needs to be entered? (e.g. `Deutsch` or `German`)? Does it match with thee free Text field of language? I could not find in the Documentation what needs to be written down in the free field that it matches. Is it case sensitive? Can we exchange the free field with a language list which can be translated?

**Comment by mouse-reeve** [comment]
That field was only half thought out -- it determines which editions are prioritized for ranking editions based on relevance. It will probably work best if the language is listed in English, but the language field in the edition table is a free text field, so it's really quite messy. I believe there has been some talk about how to make that field selectable and localized, which would be a huge improvement, but is complicated by the fact that it might need to be able to support arbitrary input as well (for cases where the language isn't in the list)

**Reply by jaschaurbach** [reply]
Easiest would be to provide a list with all languages which have an ISO-Code and make the field for the edition able to have more than one value from the list (for bi-lingual or even more-lingual editions)
